### PR TITLE
refactor: Update Dataworker types for SDK bump

### PR DESCRIPTION
**Pull request by pxrl**
The SDK no longer bundles realizedLpFeePct or relayerFeePct into the (V3)DepositWithBlock type, so make the necessary changes in the dataworker. While here, rename `realizedLpFeePct` -> `lpFeePct` to keep the code slightly more compact and concise.

### CODE REVIEW

**Comment by pxrl** [comment]
Currently marked WIP because this requires a new SDK version, which hasn't yet been built yet (subject to this PR: https://github.com/across-protocol/sdk-v2/pull/614).

File: src/clients/BundleDataClient.ts [hunk 1]
```diff
@@ -532,9 +532,9 @@ export class BundleDataClient {
     }
 
     // Process fills now that we've populated relay hash dictionary with deposits:
-    const validatedBundleV3Fills: (V3FillWithBlock & { quoteTimestamp: number })[] = [];
-    const validatedBundleSlowFills: V3DepositWithBlock[] = [];
-    const validatedBundleUnexecutableSlowFills: V3DepositWithBlock[] = [];
+    const validatedBundleV3Fills: (V3FillWithBlock & { quoteTimestamp: number; realizedLpFeePct?: BigNumber })[] = [];
```
Comment (nicholaspai) [inline src/clients/BundleDataClient.ts:535]: I don't think these should have lp fees. If they do, that's a mistake but we shouldn't be using them. On line 868 we compute lp fees for all of these

File: src/interfaces/BundleData.ts [hunk 2]
```diff
@@ -29,12 +29,12 @@ export type BundleFillsV3 = {
 
 export type BundleExcessSlowFills = {
   [destinationChainId: number]: {
-    [destinationToken: string]: interfaces.DepositWithBlock[];
+    [destinationToken: string]: (interfaces.DepositWithBlock & { realizedLpFeePct: BigNumber })[];
```
[the remark this answers is not in the document]
Comment (nicholaspai) [reply src/interfaces/BundleData.ts:32]: btw, note how we name this `lpFeePct` in BundleFillV3. Should we do the same?

File: src/clients/BundleDataClient.ts (hunk 1 above)
Comment (pxrl) [reply src/clients/BundleDataClient.ts:535]: V3FillWithBlock just had `realizedLpFeePct?` removed, so this PR is fwiw just maintaining the status quo by reinstating it locally. I'll circle back to exactly why we need it - tsc does complain if it's not there.

File: src/interfaces/BundleData.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (pxrl) [reply src/interfaces/BundleData.ts:32]: Yeah I do prefer `lpFeePct` :+1:

[the remark this answers is not in the document]
Comment (pxrl) [reply src/interfaces/BundleData.ts]: Implemented: https://github.com/across-protocol/relayer-v2/pull/1368/commits/f2767c3b71b60b3690dc9ac5c349d33c4f7d318b.

File: src/clients/BundleDataClient.ts (hunk 1 above)
Comment (pxrl) [reply src/clients/BundleDataClient.ts]: Cleaned up here: https://github.com/across-protocol/relayer-v2/pull/1368/commits/f2767c3b71b60b3690dc9ac5c349d33c4f7d318b.

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**